Automattic/WP-Job-Manager
# Add/display job category

Fixes #799 

#### Changes proposed in this Pull Request:
Added a setting in the Job Listings to be able to turn on or off showing of the job category
Made a few changes on the content-single-job_listing-company.php and wp-job-manager-template.php to allow the categories to show on the frontend.

#### Testing instructions:
On the Job Listing Settings turn on the Categories 
On the Job Listing Settings turn on the Display Job Category
Add a job with a category or multiple, it will appear on the Individual Job Page


#### Proposed changelog entry for your changes: Added ability to display job categories

NOTE: I want to improve this by allowing the categories to link to other jobs from the same category.

<img width="1267" alt="screenshot 2018-11-18 at 17 25 13" src="https://user-images.githubusercontent.com/36240423/48675986-32661380-eb58-11e8-958a-fb879efbfdc6.png">
<img width="1399" alt="screenshot 2018-11-18 at 17 25 28" src="https://user-images.githubusercontent.com/36240423/48675987-32661380-eb58-11e8-9e65-b56a065ee48a.png">
<img width="1272" alt="screenshot 2018-11-18 at 17 27 04" src="https://user-images.githubusercontent.com/36240423/48675988-32feaa00-eb58-11e8-9ca3-fc0bfb4194ba.png">
